packages/extension/src/components/DaDndMessage.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div class="dnd-message-bar">
+    <p class="do-not-disturb">Do not disturb.</p>
```

Comment:
Since this is a very simple component you don't really need the classes both for the `p` and `a` elements.
Please remove them :)